api/src/main/java/org/pmiops/workbench/api/CohortBuilderController.java

```diff
@@ -140,7 +140,12 @@
 
     @Override
     public ResponseEntity<CriteriaListResponse> getCriteriaTreeQuickSearch(String type, String value) {
+        String nameOrCode = value + "*";
```

Comment:
What if somebody enters in the value string... does that need to be escaped somehow?

Reply:
I need to get together with John and see what we want he functionality to be. We may have some validation in the client that will handle this? Keep in mind that this will morph over time as this is our first pass at this functionality in the cohort builder

Reply:
OK. From here:

https://stackoverflow.com/questions/26507087/escape-string-for-use-in-mysql-fulltext-search

it sounds like basically we should remove (or not allow) any non-word characters.